misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -1975,6 +1976,19 @@ public XMLComment transform(XMLComment xMLComment) {
                 .apply();
     }
 
+    @Override
+    public XMLCDATANode transform(XMLCDATANode xmlcdataNode) {
+        Token cDATAStart = formatToken(xmlcdataNode.cDATAStart(), 0, 0);
```

Comment:
```suggestion
        Token cdataStart = formatToken(xmlcdataNode.cdataStart(), 0, 0);
```
We seem to have used simple letters for abbreviations. e.g.  
https://github.com/ballerina-platform/ballerina-lang/blob/92ebbcd89ec81d1a18b550727e05f808bad304d8/compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json#L3681


Following is about Microsoft guidelines on writing camel case with abbreviations: https://stackoverflow.com/a/15526526/10218426

WDYT?

Reply:
+1 will update